modules/javafx.controls/src/main/java/javafx/scene/control/TableColumn.java

```diff
@@ -132,7 +131,7 @@
  * @see TablePosition
  * @since JavaFX 2.0
  */
-public class TableColumn<S,T> extends TableColumnBase<S,T> implements EventTarget {
+public class TableColumn<S,T> extends TableColumnBase<S,T> {
```

Comment:
This class is part of the public API, so this change must be reverted. The list of implemented classes shows up in the class file and in the javadoc. I suspect that this might be a compatible change, but would need to double-check to be certain. In any case, I don't want this change made.

Reply:
I checked, and yes it does produce a different result:

Current:

```
public class TableColumn<S,T>
extends TableColumnBase<S,T>
implements EventTarget
```

After proposed change:

```
public class TableColumn<S,T>
extends TableColumnBase<S,T>
```

Additionally, the class file is different. The signature for the class stored in the class file has `implements EventTarget` before this change, and not after . I'm not saying it's incompatible, but it is out of scope for this sort of global cleanup.

Reply:
How curious. I would have thought that the compiler would remove multiple inheritance paths and the classfile would be the same. As for the javadocs, it seems like the wrong behavior to me. I will try to ask on the mailing list if this is intentional.

I don't mind reverting it for this PR.